sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/screens/TextScreen.kt

```diff
@@ -0,0 +1,78 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.hmrc.sample_compose_fragments.presentation.screens
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import uk.gov.hmrc.components.compose.atom.heading.Heading3
+import uk.gov.hmrc.components.compose.atom.heading.Heading4
+import uk.gov.hmrc.components.compose.atom.heading.Heading5
+import uk.gov.hmrc.components.compose.extensions.HtmlText
+import uk.gov.hmrc.components.compose.ui.theme.CustomHmrcTypography
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme
+import uk.gov.hmrc.components.compose.ui.theme.hmrc_spacing_16
+import uk.gov.hmrc.sample_compose_components.R
+
+@Composable
+fun TextScreen() {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth().padding(hmrc_spacing_16).verticalScroll(rememberScrollState())
+    ) {
+        Heading3(text = "Heading3 text")
+        Text(
+            text = "H3 text", style = CustomHmrcTypography.h3,
+            modifier = Modifier.padding(top = hmrc_spacing_16), color = HmrcTheme.colors.hmrcBlack
+        )
+        Heading4(text = "Heading4 text", modifier = Modifier.padding(top = hmrc_spacing_16))
+        Text(
+            text = "H4 text", style = CustomHmrcTypography.h4,
+            modifier = Modifier.padding(top = hmrc_spacing_16), color = HmrcTheme.colors.hmrcBlack
+        )
+        Heading5(text = "Heading5 text", modifier = Modifier.padding(top = hmrc_spacing_16))
+        Text(
+            text = "H5 text", style = CustomHmrcTypography.h5,
+            modifier = Modifier.padding(top = hmrc_spacing_16), color = HmrcTheme.colors.hmrcBlack
+        )
+        Text(
+            text = "Bold text", style = CustomHmrcTypography.h6,
+            modifier = Modifier.padding(top = hmrc_spacing_16), color = HmrcTheme.colors.hmrcBlack
+        )
+        Text(
+            text = "Body text", style = CustomHmrcTypography.body,
+            modifier = Modifier.padding(top = hmrc_spacing_16), color = HmrcTheme.colors.hmrcBlack
+        )
+        Text(
+            text = "Info text", style = CustomHmrcTypography.info,
+            modifier = Modifier.padding(top = hmrc_spacing_16), color = HmrcTheme.colors.hmrcGrey1
+        )
+        HtmlText(
+            text = stringResource(id = R.string.text_link),
+            modifier = Modifier.padding(top = hmrc_spacing_16)
+        )
+        Text(
+            text = "Error text", color = HmrcTheme.colors.hmrcRed,
+            style = CustomHmrcTypography.body, modifier = Modifier.padding(top = hmrc_spacing_16)
+        )
```

Comment:
I think we can make `errorText` one of the `HmrcTypography` types and declare it explicitly.